website/docs/compute/quickstart/client-libraries/nodejs.md

```diff
@@ -1,6 +1,6 @@
 # Using Compute from JavaScript or TypeScript
 
-Use this library to easily interact with Compute's APIs.
+Use [this](https://suborbital.github.io/docs/compute/quickstart/client-libraries/nodejs) library to easily interact with Compute's APIs.
```

Comment:
This won't be the final domain, it should be a relative link like `/compute/quickstart/client-libraries/nodejs.md`